quiz_app/questions/views.py

```diff
@@ -1,5 +1,6 @@
 import json
```

Comment:
Now it's time to use class based views in here (I just forgot to mention it before xD).
After that you can just used `LoginRequiredMixin` instead of a decorator :)